services/paymaster/src/constraint/mod.rs

```diff
@@ -73,26 +76,18 @@ pub struct ContextualDomainKeys {
     pub sponsor: Pubkey,
 }
 
-fn instruction_matches_program(
-    transaction: &VersionedTransaction,
-    instruction_index: usize,
-    program_to_match: &Pubkey,
-) -> anyhow::Result<bool> {
-    let instruction = transaction.message.instructions().get(instruction_index);
-    if let Some(instruction) = instruction {
-        let static_accounts = transaction.message.static_account_keys();
-        let program_id = instruction.program_id(static_accounts);
-        if program_id == program_to_match {
-            return Ok(true);
-        }
-    } else {
-        anyhow::bail!("Instruction index {instruction_index} out of bounds");
-    }
+impl VariationOrderedInstructionConstraints {
+    const IGNORED_PROGRAMS: &[Pubkey] =
+        &[solana_compute_budget_interface::id(), TOLLBOOTH_PROGRAM_ID];
```

Comment:
I'm going to take this offline but I feel uneasy about the idea of adding a set of instructions that are globally unconstrained